tests/js/spec/components/avatar.spec.jsx

```diff
@@ -26,7 +26,7 @@ describe('Avatar', function() {
         },
       });
       let avatar = mount(<Avatar user={user} />);
-      expect(avatar.find('.avatar')).toHaveLength(1);
+      expect(avatar.find('span.avatar')).toHaveLength(1);
```

Comment:
can we look for `StyledBaseAvatar` here? coupling tests with presentation classes worries me, especially in situations like `Button` where we try to decouple things.

Reply:
I think my reasoning here for using a presentational class, is that `Avatar` still has legacy CSS that requires the class name (and the test specifically is for checking the class name).